i18n: Update Japanese translation

kisaragi-hiu opened the pull request:
Trying my best to match Japanese KDE.

- Update new strings with lupdate. ...except the (Wayland) string is left as-is without changing its context.
- Try my best to translate the new strings.
- Use "Caps Lock" for "Caps Lock". This is existing convention, already used in eg. kdeplasma-addons, plasma-desktop, and plasma5support.
- ~~Use ユーザー名 for username. This is, for instance, recorded in goo辞書 while ユーザ名 isn't.~~ ~~On (personal) review Japanese KDE convention is ユーザ名 ([like ユーザ会](https://jp.kde.org/)); this PR now matches that.~~ This PR now uses ユーザー again per feedback.

kisaragi-hiu committed:
Update Japanese translation

Trying my best to match Japanese KDE.

- Update new strings with lupdate.
  ...except the (Wayland) string is left as-is without changing its context.
- Try my best to translate the new strings.
- Use "Caps Lock" for "Caps Lock". This is existing convention, already
  used in eg. kdeplasma-addons, plasma-desktop, and plasma5support.

phanect commented on data/translations/ja.ts:
```diff
@@ -69,5 +69,49 @@
+    </message>
+    <message>
+        <source>Show password</source>
+        <translation>パスワードを表示する</translation>
```
While 表示する is still OK, I think 表示 is a bit more simple and natural.

```suggestion
        <translation>パスワードを表示</translation>
```

phanect commented on data/translations/ja.ts:
```diff
@@ -69,5 +69,49 @@
+    </message>
+    <message>
+        <source>Hide password</source>
+        <translation>パスワードを隠す</translation>
```
While 隠す makes sense, I think 非表示 is better to express contrast to 表示.

```suggestion
        <translation>パスワードを非表示</translation>
```

phanect commented on data/translations/ja.ts:
```diff
@@ -69,5 +69,49 @@
+    </message>
+    <message>
+        <source>Password change aborted because maximum tries reached</source>
+        <translation>最大試行回数に達しましたため、パスワード変更は中止しました</translation>
```
For messages on the software UI, 「達しましたため」 is a bit too formal.

```suggestion
        <translation>最大試行回数に達したため、パスワード変更は中止しました</translation>
```

phanect replied:
Is this text used anywhere?
I quickly searched this repository and suspect this text is no longer used.

kisaragi-hiu replied:
These are from [TextConstants.qml](https://github.com/sddm/sddm/blob/2d27436bab48eb49e4607ab073ab7ad1a98b9350/components/common/TextConstants.qml#L4), and themes use it for common strings. For instance, [sddm-astronaut-theme](https://github.com/search?q=repo%3AKeyitdev%2Fsddm-astronaut-theme%20textconstants&type=code); [other search results](https://github.com/search?q=sddm+theme+textconstants+NOT+language%3Axml&type=code)

phanect replied:
Got it. The text is used in another repositories. Thanks!

phanect commented on data/translations/ja.ts:
```diff
@@ -69,5 +69,49 @@
+    </message>
+    <message>
+        <source>Enter your username</source>
+        <translation>ユーザ名を入力</translation>
```
While we, Japan KDE Users Group, have not renamed our organization name from 日本 KDE **ユーザ**会 to **ユーザー会** in Japanese yet, the recent trend is **ユーザー** (with 長音符: "ー") because expression without 長音符 (ー) is (or was) only used in software and some other specific industries.

```suggestion
        <translation>ユーザー名を入力</translation>
```

phanect commented on data/translations/ja.ts:
```diff
@@ -5,14 +5,14 @@
     <message>
         <source>Press to login</source>
-        <translation type="unfinished"></translation>
+        <translation>押してログイン</translation>
```
@kisaragi-hiu
It sounds a bit unnatural, but I'm not sure how to make it natural because I don't know the context...
I found it is used [here](https://github.com/sddm/sddm/blob/develop/components/2.0/PictureBox.qml#L125), but not sure how it is displayed. (Screenshot is much appreciated if you have)

kisaragi-hiu replied:
It's used in the default Maui theme, shown for users that do not have passwords.

![螢幕截圖_20250126_010258](https://github.com/user-attachments/assets/7b849247-6086-4b6a-b468-6b209d95b4c1)

phanect replied:
Thank you.
I could not find the best translation, but I suggest adding ここを.
The new Japanese translation literally means "Press **here** to login."

```suggestion
        <translation>ここを押してログイン</translation>
```

kisaragi-hiu replied:
Applied in [`2aaa80d` (#2036)](https://github.com/sddm/sddm/pull/2036/commits/2aaa80da6995ce3f57d286d4fde92851264779f6)

phanect commented on data/translations/ja.ts:
```diff
@@ -69,5 +69,49 @@
+    </message>
+    <message>
+        <source>New password change round! Please input current password again!</source>
+        <translation>新しいパスワード変更ラウンドです! 今のパスワードをもう一度入力してください!</translation>
```
@kisaragi-hiu
"パスワード変更ラウンド" sounds a bit unnatural, but I'm not sure how to make it natural because I don't know the context.
Do you know where this text is used?
(Screenshot is much appreciated if you have)

phanect replied:
Is this text used anywhere?
I quickly searched this repository and suspect this text is no longer used.

kisaragi-hiu replied:
I'm not sure this is used anywhere either. Searches for the text or [the key](https://github.com/sddm/sddm/blob/2d27436bab48eb49e4607ab073ab7ad1a98b9350/components/common/TextConstants.qml#L50) on GitHub just land back at [TextConstants.qml](https://github.com/sddm/sddm/blob/2d27436bab48eb49e4607ab073ab7ad1a98b9350/components/common/TextConstants.qml) in this repository.

The name of the key suggests it might've been meant for [PAM lockout (max tries)](https://wiki.archlinux.org/title/Security#Lock_out_user_after_three_failed_login_attempts) at some point, but then the text just doesn't make sense.

phanect replied:
Then I think we can keep your current translation for now.
The current translation is not meaningful enough, but it is much better than showing the wrong information by trying a more natural translation.

phanect replied:
Or we have another option to remove this Japanese translation for now.

Vogtinator replied:
FWICT it's used in the stale `expired-password` branch and was added such that themes can use some SDDM provided translations. I don't think it's used at all ATM.

kisaragi-hiu replied:
I've removed these two strings from the PR.

phanect commented on data/translations/ja.ts:
```diff
@@ -69,5 +69,49 @@
     </message>
+    <message>
+        <source>Please enter a password!</source>
+        <translation>パスワードを入力してください!</translation>
```
In the Japanese context, adding an exclamation mark (!) here sounds like a bit too strong.
To avoid letting users feel offended or blamed, I suggest removing the exclamation mark.

If possible, I want to know why the developers added the original English text to the exclamation mark for better translation.

```suggestion
        <translation>パスワードを入力してください</translation>
```

phanect reviewed (changes requested):
I'm a native Japanese speaker who got a review request for this PR via the KDE Japanese community mailing list.
I proofread the translations by @kisaragi-hiu, who is not a native Japanese speaker.

phanect commented on data/translations/ja.ts:
```diff
@@ -23,7 +23,7 @@
     <message>
         <source>Warning, Caps Lock is ON!</source>
-        <translation>注意: Caps ロックがオンになっています!</translation>
+        <translation>注意: Caps Lock が有効です!</translation>
```
I want to revert this translation except for ロック → Lock.
The reason why I reverted the most part:

- I quickly googled and *ON*/*OFF* (*オン*/*オフ*) looks like a bit more popular expression than *有効*/*無効* in Japanese. In addition, the original text is not *enabled* but *ON*, I think *オン* is a better translation than *有効*.
- I think になっています is better to express the current *status* of Caps Lock than です.

I also removed the exclamation mark (!) because it sounds a bit too emphasized in Japanese.

```suggestion
        <translation>注意: Caps Lock がオンになっています</translation>
```

kisaragi-hiu committed:
Apply suggestions from code review

Co-authored-by: Jumpei Ogawa <phanective@gmail.com>

kisaragi-hiu committed:
Apply suggestions from review

- Add ここを as per review
- Leave out unused strings from the expired-password branch

Co-authored-by: Jumpei Ogawa <phanective@gmail.com>

phanect reviewed (approved):
Thanks, looks good to me.
